steps/waldo-upload/0.2.0/step.yml

```diff
@@ -0,0 +1,90 @@
+title: Waldo Upload
+summary: |
+  Uploads a build to Waldo.
+description: |
+  Uploads a build to Waldo.
+website: https://github.com/waldoapp/bitrise-step-waldo-upload
+source_code_url: https://github.com/waldoapp/bitrise-step-waldo-upload
+support_url: https://github.com/waldoapp/bitrise-step-waldo-upload/issues
+published_at: 2020-09-21T10:28:59.961629-04:00
+source:
+  git: https://github.com/waldoapp/bitrise-step-waldo-upload.git
+  commit: a1dc3ed369c0fcb3a7d20fa7bb2070dec9f61fba
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- ios
+- android
+- xamarin
+- react-native
+- cordova
+- ionic
+- flutter
+type_tags:
+- test
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: curl
+  apt_get:
+  - name: curl
+is_requires_admin_user: false
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- build_path: null
+  opts:
+    description: |
+      A `.app`, `.ipa`, or `.apk` path.
+
+      Typically this path is generated by a previous build Step in your
+      Workflow. For example:
+
+      - the **Xcode build for simulator** Step generates a `.app` at `$BITRISE_APP_DIR_PATH`
+      - the **Xcode Archive & Export for iOS** Step generates a `.ipa` at `$BITRISE_IPA_PATH`
+      - the **Android Build** Step generates a `.apk` at `$BITRISE_APK_PATH`
+    is_required: true
+    summary: A `.app`, `.ipa`, or `.apk` path.
+    title: Build path
+- opts:
+    description: |
+      Each application registered in Waldo is associated with a unique upload
+      token.
+    is_required: true
+    is_sensitive: true
+    title: Waldo upload token
+  upload_token: $WALDO_UPLOAD_TOKEN
+- include_symbols: "no"
+  opts:
+    category: Symbols
+    description: |
+      **iOS only.** Attempts to find the symbols associated with the
+      specified `.app` build path.
+
+      This input is _ignored_ if a symbols path is specified.
+    is_required: false
+    title: Include symbols with the build upload
```

Comment:
This title is misleading for me. Taking description into account it should be rather called like "Search for symbols". It sounds preposterous that "including symbols" is ignored when symbols path is specified.

Reply:
I took the name from the underlying script that this step is wrapping. Normally, when you build an app for simulator, Xcode leaves the `.dSYM` bundle right next to the `.app` bundle. In that case, `include_symbols` makes sense. Unfortunately, the Bitrise `Xcode build for simulator` step _copies_ the `.app` bundle from some internal location to a more public one (pointed to by `$BITRISE_APP_DIR_PATH`) but does _not_ do the same for the `.dSYM` bundle (and thus `$BITRISE_DSYM_PATH` is useless). I used `include_symbols` to name this input in the hope that eventually the `Xcode build for simulator` step is fixed.

If you insist, I can rename the input to `find_symbols` or something similar.